src/inc/corinfo.h

```diff
@@ -864,6 +864,7 @@ enum CorInfoMethodRuntimeFlags
     CORINFO_FLG_BAD_INLINEE         = 0x00000001, // The method is not suitable for inlining
     CORINFO_FLG_VERIFIABLE          = 0x00000002, // The method has verifiable code
     CORINFO_FLG_UNVERIFIABLE        = 0x00000004, // The method has unverifiable code
+    CORINFO_FLG_TIER0_TO_TIER1      = 0x00000008, // The JIT decided to switch from tier 0 to tier 1 for this method
```

Comment:
Although it doesn't matter much now, I wanted to call out that the JIT is switching to tier 1 because the VM is going to treat this method as though it is at tier 1, whatever that means in the future. I think the "tier 0" part can be assumed or taken care of by the VM but the "tier 1" part may be interesting in the future. What do you think about `CORINFO_FLG_SWITCHED_TO_TIER1`?

Reply:
That seems reasonable